tasks/asserts.yml

```diff
@@ -137,6 +137,22 @@
 ###
 ### Validate security setting
 ###
+- name: "({{ cluster.name }}) ensure kops_dockerconfig_creds format"
+  assert:
+    that:
+      - item.user is defined
+      - item.user | length > 0
+      - item.token is defined
+      - item.token | length > 0
+    msg: "kops_dockerconfig_creds should include user and token and validated in dockerhub"
+  with_items:
+    - >-
+      {%- if kops_dockerconfig_creds is defined -%}
```

Comment:
Where is `kops_dockerconfig_creds` defined? And where can I find documentation about this?

Reply:
Updated it in README